apps/els_lsp/src/els_erlfmt_ast.erl

```diff
@@ -0,0 +1,1290 @@
+%% Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
Why forking it, instead of adding it as a dependency?

Reply:
Ah, saw the branch now. What's the plan long term for this?

Reply:
I hope that branch will eventually be merged and erlfmt_ast be a part of erlfmt and we can remove this temporary copy from erlang_ls

Reply:
@michalmuskala Any insights on that?